src/com/company/Main.java

```diff
@@ -3,7 +3,91 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+	// Задача 1
+        System.out.println("Задача 1");
+        byte clientOS=0;
+        if (clientOS==0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+        if (clientOS==1) {
```

Comment:
```suggestion
        }else if (clientOS==1) {
```
лучше объядинить в один оператор, поскольку условия взаимоисключающие